README.md

```diff
@@ -1174,3 +1175,14 @@ An example structure:
         - getter and setter for same property should appear consecutively
     - public interface methods
     - private interface methods
+
+### Macros
+
+- Inside Objective-C source, use `typeof()`.  When using the Clang compiler, or using gcc AND `GCC_ENABLE_ASM_KEYWORD=YES`, it's equivalent to the underscored versions (`__typeof()` and `__typeof()__`).
+
+- When defining a macro that includes a parameter, parenthesize any use of the parameter.
```

Comment:
It would be nice to provide an example or link to an article explaining why this is good practice (e.g. passed values that contain control flow characters).